include/odp/api/spec/packet_flags.h

```diff
@@ -23,87 +23,114 @@ extern "C" {
 #include <odp/api/packet.h>
 
 /** @addtogroup odp_packet
- *  Boolean operations on a packet.
+ *  Operations on packet metadata flags.
  *  @{
  */
 
 /**
- * Check for packet errors
+ * Check for all errors in packet
  *
- * Checks all error flags at once.
+ * Check if packet parsing have found any errors in the packet. The level of
+ * error checking depends on parse configuration (e.g. included layers and
```

Comment:
"the parse configuration" would be better here

Reply:
OK.